.gitignore

```diff
@@ -78,6 +78,9 @@ target/
 # Jupyter Notebook
 .ipynb_checkpoints
 
+# Pycharm IDE
+.idea
```

Comment:
It's already been added, but it's commented out (dunno why). Remove this and uncomment the other one.